src/app/features/auth/resetpassword/ResetPasswordForm.tsx

```diff
@@ -20,20 +22,17 @@ export const ResetPasswordForm: FC<{
   >
     {formikProps => (
       <Form noValidate>
-        <FormControl
-          label="Email"
-          name="email"
-          type="email"
-          formikProps={formikProps}
-          placeholder="user_name@mail.com"
-        />
-        <Button
-          block
-          color="primary"
-          type="submit"
-          disabled={formikProps.isSubmitting}
-        >
-          Send Link
+        <div hidden={isSuccess}>
```

Comment:
Could you please investigate, can we use conditional rendering here instead of hidden?